setup/inject_SELinux_check/test.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+BEAKERLIB_SCRIPT=/usr/share/beakerlib/logging.sh
+
+rlJournalStart
+
+    rlPhaseStartTest
+
+        #for testing purposes
+        rlRun "git clone --branch test_AVC https://github.com/Koncpa/keylime-selinux.git"
+        rlRun "pushd keylime-selinux"
+        rlRun "make -f /usr/share/selinux/devel/Makefile keylime.pp"
+        rlRun "semodule -i keylime.pp"
+        rlRun "restorecon -R /"
+        rlRun "popd"
+
+        rlLogInfo "Injecting the test code into rlPhaseStartTest"
+        rlRun "cp $BEAKERLIB_SCRIPT $BEAKERLIB_SCRIPT.pre_injection_backup" 0 "Making backup of $BEAKERLIB_SCRIPT"
+        if grep -q "__INTERNAL_TIMESTAMP_AVC" $BEAKERLIB_SCRIPT; then
+            rlRun "echo 'It was already injected into' $BEAKERLIB_SCRIPT"
+        else
+            rlRun "sed -i '/rlPhaseStart()/a\    STAMP_SETUP=\`LC_ALL=en_US.UTF-8 date \"+%x %T\"\`\n    __INTERNAL_TIMESTAMP_AVC=\$STAMP_SETUP\n    export __INTERNAL_TIMESTAMP_AVC\n' $BEAKERLIB_SCRIPT"
```

Comment:
I guess you can avoid using `STAMP_SETUP` and assign date to __INTERNAL_TIMESTAMP_AVC directly.